app/src/main/java/ch/epfl/sdp/entity/Entity.java

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.sdp.entity;
+
+/**
+ * Base interface for all entities.
+ */
+public interface Entity {
+
+    /**
+     * Method to get the type of the object we want to display
+     * @return an EntityType which is an enum of types
+     */
+    EntityType getEntityType();
+}
```

Comment:
This file should be deleted, we are not using this anymore in the master branch.